java/api/src/main/java/io/ray/api/Ray.java

```diff
@@ -85,15 +85,15 @@ public static synchronized void shutdown() {
   }
 
   /**
-   * Get a list of objects by RayObjects from the object store.
+   * Get a list of objects by ObjectRefs from the object store.
    *
-   * @param objectList A list of RayObject to get.
+   * @param objectList A list of object refs .
```

Comment:
```suggestion
   * @param objectList A list of object references .
```
same for other places.